config/scheduler-chromeos.yaml

```diff
@@ -50,60 +50,106 @@ _anchors:
       type: lava
       name: lava-collabora
 
-  test-job-mediatek: &test-job-mediatek
+  test-job-arm64-mediatek: &test-job-arm64-mediatek
     <<: *lava-job-collabora
     event:
       channel: node
-      name: kbuild-gcc-10-arm64-chromeos-mediatek
+      name: kbuild-gcc-10-arm64
```

Comment:
Those jobs won't run: the chromebook platform configs require the `arm64-chromebook` fragment to be used, which is not the case for this kbuild job.

I'd suggest you simply modify `kbuild-gcc-10-arm64` (defined in the main pipeline config) to use said fragment (the x86 one already uses the `x86-board` fragment, so all good there).

Reply:
I don't think it's a problem, what this fragment basically does is:
* enable a bunch of drivers (and make sure some of those are built-in), most of which won't even be probed on other devices
* ensure NFS support is built-in as well, which can't harm

I don't think this would affect other boards, except maybe the `CONFIG_DRM...=y` options if I'm being pessimistic, but hey, there are very similar stuff in `x86-board` as well, so let's enable this fragment and revert if it causes any harm.

Reply:
Ack, thanks for the feedback! I added the fragment to `kbuild-gcc-10-arm64`.